integration/stability/bind_mount_linux.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Missing copyright by the author here?

Reply:
update

Reply:
Thanks. I guess you may have written this in 2018, but should this be 2019 now? :smile:

Reply:
oh, yes😂